src/index.ts

```diff
@@ -0,0 +1,37 @@
+import http = require("http");
+import fs = require("fs");
+
+const postData: http.RequestListener = (
+  request: http.IncomingMessage,
+  response: http.ServerResponse
+) => {
+  if (request.method === "POST") {
+    let body = "";
+    request.on("data", (data) => {
+      body += data;
+    });
+    request.on("end", () => {
+      const preparedData = dataToUpperCase(body) + "\n";
+      fs.appendFileSync("..\\data\\data.txt", preparedData);
+    });
+    response.end("Data updated.");
+  } else {
+    response.statusCode = 403;
+    response.statusMessage = "Forbidden activity.";
```

Comment:
You can specify error code and error message in one function res.writeHead();

Reply:
Changed